docs/api/web-contents.md

```diff
@@ -502,6 +502,21 @@ win.loadURL('http://github.com')
 
 Emitted when the devtools window instructs the webContents to reload
 
+#### Event: 'will-attach-webview'
+
+Returns:
+
+* `event` Event
+* `guest` WebContents - The contents of the guest page.
```

Comment:
Would it be better not to provide `guest` parameter with this event, since they can destroy it with `ev.preventDefault`.  We can add another event `did-attach-webview` where we can provide the `webContents` associated as a parameter and its safer to attach any listeners there. `will-attach-webview` should just be used for modifying any parameters used for webview creation , while `did-attach-webview` can be used to interact with it.